mipengine/node/monetdb_interface/common.py

```diff
@@ -7,19 +7,20 @@
 from mipengine.node.node import config
 from mipengine.node.tasks.data_classes import ColumnInfo
 from mipengine.node.tasks.data_classes import TableSchema
+from mipengine.utils.verify_identifier_names import sql_injections_defender
```

Comment:
Είμαι ανοικτός σε προτάσεις για καλύτερο όνομα απο το sql_injections_defender.

Reply:
`validate_idnames`?

Reply:
The package should probably be something like `validate_input `instead of `verify_identifier_names `and instead of `sql_injections_defender `something `validate_proper_string_input` .

Reply:
`validate_identifier_names` or `validate_idnames` for short describes exactly what it does. `validate_string_input` (proper is obvious) is less informative in my opinion.